x-pack/plugins/cases/public/containers/use_delete_comment.test.tsx

```diff
@@ -5,94 +5,110 @@
+
+  (useToasts as jest.Mock).mockReturnValue({ addSuccess, addError });
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
   it('init', async () => {
-    await act(async () => {
-      const { result, waitForNextUpdate } = renderHook<string, UseDeleteComment>(() =>
-        useDeleteComment()
-      );
-      await waitForNextUpdate();
-      expect(result.current).toEqual({
-        isError: false,
-        deleteComment: result.current.deleteComment,
-      });
+    const { result } = renderHook<string, UseDeleteComment>(() => useDeleteComment(), {
+      wrapper,
     });
+
+    expect(result.current).toBeTruthy();
   });
 
   it('calls deleteComment with correct arguments - case', async () => {
     const spyOnDeleteComment = jest.spyOn(api, 'deleteComment');
 
-    await act(async () => {
-      const { result, waitForNextUpdate } = renderHook<string, UseDeleteComment>(() =>
-        useDeleteComment()
-      );
-      await waitForNextUpdate();
+    const { waitForNextUpdate, result } = renderHook<string, UseDeleteComment>(
+      () => useDeleteComment(),
+      {
+        wrapper,
+      }
+    );
 
-      result.current.deleteComment({
+    act(() => {
+      result.current.mutate({
         caseId: basicCaseId,
         commentId,
       });
-      await waitForNextUpdate();
-      expect(spyOnDeleteComment).toBeCalledWith({
-        caseId: basicCaseId,
-        commentId,
-        signal: expect.any(AbortSignal),
-      });
-      expect(result.current.isError).toBe(false);
+    });
+
+    await waitForNextUpdate();
+
+    expect(spyOnDeleteComment).toBeCalledWith({
+      caseId: basicCaseId,
+      commentId,
+      signal: expect.any(AbortSignal),
     });
   });
 
   it('refreshes the case page view after delete', async () => {
-    await act(async () => {
-      const { result, waitForNextUpdate } = renderHook<string, UseDeleteComment>(() =>
-        useDeleteComment()
-      );
-      await waitForNextUpdate();
+    const { waitForNextUpdate, result } = renderHook<string, UseDeleteComment>(
+      () => useDeleteComment(),
+      {
+        wrapper,
+      }
+    );
 
-      result.current.deleteComment({
-        caseId: basicCaseId,
-        commentId,
-      });
-      await waitForNextUpdate();
-      expect(useRefreshCaseViewPage()).toBeCalled();
+    result.current.mutate({
+      caseId: basicCaseId,
+      commentId,
     });
+
+    await waitForNextUpdate();
+
+    expect(useRefreshCaseViewPage()).toBeCalled();
   });
 
   it('sets isError when fails to delete a case', async () => {
     const spyOnDeleteComment = jest.spyOn(api, 'deleteComment');
     spyOnDeleteComment.mockRejectedValue(new Error('Not possible :O'));
 
-    await act(async () => {
-      const { result, waitForNextUpdate } = renderHook<string, UseDeleteComment>(() =>
-        useDeleteComment()
-      );
-      await waitForNextUpdate();
+    const { waitForNextUpdate, result } = renderHook<string, UseDeleteComment>(
```

Comment:
Let's add a test that we the `addToast` has been called (as in `use_get_feature_ids.test.tsx`)